apps/code-examples/src/app/code-examples/phone-field/phone-field/basic/demo.component.ts

```diff
@@ -21,15 +22,17 @@ import { SkyPhoneFieldModule } from '@skyux/phone-field';
   ],
 })
 export class DemoComponent {
-  protected phoneForm: FormGroup;
+  public phoneForm: FormGroup;
```

Comment:
Why are these changing to be `public`?

Reply:
We have been making form controls or groups public to be accessible in tests.